pkg/controller/account/byoc.go

```diff
@@ -289,9 +288,9 @@ func DetachPolicyFromRole(reqLogger logr.Logger, policy *iam.AttachedPolicy, byo
 			default:
 				reqLogger.Error(
 					aerr,
-					fmt.Sprintf(aerr.Error()),
+					aerr.Error(),
 				)
-				reqLogger.Error(err, fmt.Sprintf("%v", err))
+				reqLogger.Error(err, err.Error())
```

Comment:
nit: this will end up printing the same error twice. This could just be

```suggestion
				reqLogger.Error(err, "")
```

...to the same effect.

(Here and below.)

Reply:
Thanks!  I'm refactoring how the errors are handles in the byoc.go for another PR, and will make those changes there.